utils/cryptodev-linux/Makefile

```diff
@@ -17,6 +17,7 @@ PKG_SOURCE_URL:=https://github.com/cryptodev-linux/cryptodev-linux
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_VERSION:=a705360197260d28535746ae98c461ba2cfb7a9e
 PKG_MIRROR_HASH:=334a8f4df18e36e97459b4c6669c9873c13c4f5dae25243cf41ff70c2ce661d8
+PKG_LICENSE:=GPL-2.0+
```

Comment:
Looks like it should be only "GPL-2.0" according to content of https://github.com/cryptodev-linux/cryptodev-linux/blob/master/COPYING

Reply:
`PKG_LICENSE_FILES` should also be added.

Reply:
Sorry this was my vault!
Will fix it